src/uploaders/nip96.ts

```diff
@@ -1,32 +0,0 @@
-import type { EventTemplate, NostrEvent } from 'nostr-tools/core'
-import { readServerConfig, uploadFile } from 'nostr-tools/nip96'
-import { getToken } from 'nostr-tools/nip98'
-import type { UploadTask } from './types'
-
-export interface NIP96Options {
-  file: File
-  alt?: string
-  serverUrl: string
-  expiration?: number
-  sign?: (event: EventTemplate) => Promise<NostrEvent> | NostrEvent
-}
-
-export async function uploadNIP96(options: NIP96Options): Promise<UploadTask> {
```

Comment:
the library still should provide these functions no?

Reply:
Well, similar to the encryption functions they're now only being called by the application code. There are plenty of implementations of blossom/nip96 out there, and the impetus for this PR was that the blossom uploader no longer worked for my use case. I can leave them in if you want, but it seems better to put them in nostr-tools or somewhere else instead.